src/main/java/de/tk/opensource/privacyproxy/routing/RoutingHandler.java

```diff
@@ -83,6 +74,10 @@ public ResponseEntity<Resource> handleGenericRequestInternal(
             @Nullable final String body,
             final HttpMethod method
     ) {
+        if (method == HttpMethod.GET && body != null) {
+            return ResponseEntity.status(HttpStatus.FORBIDDEN).build();
```

Comment:
Wäre hier vlt Bad Request semantisch korrekter?

Reply:
Ich wollte hier gewollt nicht die Return Statuscodes ändern, ich nimms mit in den Major PR